libpod/oci_conmon_common.go

```diff
@@ -1313,6 +1313,9 @@ func (r *ConmonOCIRuntime) configureConmonEnv(runtimeDir string) []string {
 		if strings.HasPrefix(e, "LC_") {
 			env = append(env, e)
 		}
+		if strings.HasPrefix(e, "LANG=") {
```

Comment:
It might not be that simple. Please check to see if you also need to pass `LC_<x>` envariables. (I don't know enough to be sure). For future spelunkers, please cite appropriate references.

Reply:
Sigh.... I should refrain from code review that late in the day...